test/k8sT/Services.go

```diff
@@ -816,25 +816,25 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sServicesTest", func() {
 			}
 
 			var wg sync.WaitGroup
-			wg.Add(len(testCases) * 2)
-			for _, testCase := range testCases {
-				for _, name := range []string{be1Name, be2Name} {
-					go func(tc lrpTestCase, want string) {
+			for _, tc := range testCases {
+				pods, err := kubectl.GetPodNames(helpers.DefaultNamespace, tc.selector)
+				Expect(len(pods)).Should(BeNumerically(">", 0), "no pod exists by filter %q", tc.selector)
+				Expect(err).Should(BeNil(), "cannot retrieve pod names by filter %q", tc.selector)
```

Comment:
might be better to check err first to get slightly more useful output on failure, or was there a specific reason the order changed?